negar/gui.py

```diff
@@ -1,10 +1,17 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
+import os
 import sys
-from PyQt4.QtGui import *
-from PyQt4.QtCore import *
-from virastar import PersianEditor, add_to_untouchable
+from PyQt6.QtGui import *
+from PyQt6.QtCore import *
+from PyQt6.QtWidgets import *
+try:
+    from virastar import PersianEditor, add_to_untouchable
+except:
+    from .virastar import PersianEditor, add_to_untouchable
```

Comment:
```suggestion
from negar.virastar import PersianEditor, add_to_untouchable
```

Reply:
Actually, in my machine `from negar.viraster import ...` just loads from the installed library (`/lib/python3.10/site-packages/python_negar-0.6.6-py3.10.egg/negar`), not the local one and I made it to fix this issue! I meant when I wanted to run the `gui.py` from the `negar` directory it threw the following error:
`ModuleNotFoundError: No module named 'negar'`